dev/Repeater/ElementFactory.h

```diff
@@ -20,7 +20,7 @@ class ElementFactory :
 #pragma endregion
 
     // TODO: Bug 14901501: Figure out a better way to have reference tracking for types doing in-component derivation (e.g. RecyclingElementFactory : ElementFactory)
-    virtual HRESULT __stdcall NonDelegatingQueryInterface(const GUID& id, void** object)
+    virtual int32_t __stdcall NonDelegatingQueryInterface(const winrt::guid& id, void** object)
```

Comment:
I *think* the expectation was that this overrode the base method ... but I'm not sure why it didn't specify "override" modifier. @StephenLPeters did you take a look at the bug fix where this code was introduced to make sure we exercise that scenario here?

Reply:
Thanks to the mismatched parameters, it didn't actually override it. I wasn't sure if overriding was intended. My signature change makes it an override. I don't think it matters because

1. INonDelegatingInspectable calls NonDelegatingQueryInterface via `shim()` which will forward out to the most-derived class via CRTP anyway, and
2. The mismatched declaration shadowed the original one, which is as good as overriding when called via CRTP.

But finding a way to exercise this code would help me sleep at night.